src/authAgents/SessionAuthAgent.js

```diff
@@ -273,5 +286,24 @@ define(["structures/CAPIError", "storages/LocalStorage"], function (CAPIError, L
         this._storage.removeItem(SessionAuthAgent.KEY_CSRF_TOKEN);
     };
 
+    /**
+     * Stores the session information in the storage
+     *
+     * @method _storeSession
+     * @param {Object} session an object describing the session
+     * @param session.name {String} the name of the session
+     * @param session.identifier {String} the identifier of the session
+     * @param session.href {String} the resource uri to refresh the session
+     * @param session.csrfToken {String} the CSRF Token associated with the
+     * session
+     * @protected
+     */
+    SessionAuthAgent.prototype._storeSession = function (session) {
```

Comment:
It's not really about storing the data of the session, but rather its meta data, so maybe call it storeSessionInfo or MetaData or something.